Lib/unittest/test/testmock/testsealable.py

```diff
@@ -199,6 +199,7 @@ def ban(self):
                 self.assertIsInstance(foo.Baz, mock.MagicMock)
                 self.assertIsInstance(foo.Baz.baz, mock.NonCallableMagicMock)
                 self.assertIsInstance(foo.Baz.ban, mock.MagicMock)
+                self.assertIsInstance(foo.bar2(), mock.Mock)
```

Comment:
This assertion appears redundant, given the one on line 198?

Reply:
The line 198 does not call the method, and was not triggering the AttributeError without the change in this PR. The new line 202 does trigger AttributeError without this change.